docs/12-system-tests.md

```diff
@@ -0,0 +1,164 @@
+```
+
+This will generate everything for you including:
+
+* An S3 Bucket containing all the necessary jars
+* ECR repositories for ingest, compaction and system test images
+* The Sleeper properties file
+* Random test data in the `system-test` table.
+
+Once generated, it deploys Sleeper using CDK.
+
+This test has an instance properties file next to the deploy script called `system-test-instance.properties`.
+When running the deploy script, the `scripts/test/deployAll` directory is scanned for `table.properties`,
+`tags.properties`, and `schema.properties` files. If they are found, they are picked up and used by the test.
+If they are not present, then the template files in `scripts/templates` are used. Note that properties in these
+files with the value `changeme` will be overwritten by the script.
+
+There's an optional stacks property which determines which components will be deployed - you may want to customise this
+to experiment with different stacks.
+
+There are also properties specific to system tests, which can also be changed in `system-test-instance.properties`.
+These can be used to configure how much test data will be generated, and how much that data can vary.
+
+All resources are tagged with the tags defined in the file `scripts/templates/tags.template`, or a `tags.properties`
+file placed next to the `system-test-instance.properties`.
+
+You can get a report of your instance by running:
+
+```bash
+./scripts/test/systemTestStatusReport.sh
+```
+
+Finally, when you are ready to tear down the instance, run:
+
+```bash
+./scripts/test/tearDown.sh
+```
+
+This will check the `scripts/generated` folder for the instance to tear down. That gets created during deployment, or
+with `scripts/utility/downloadConfig.sh`. Alternatively, you can skip reading the generated folder by giving an instance
+ID:
+
+```bash
+./scripts/test/tearDown.sh <instance-id>
+```
+
+This will remove your deployment, including any ECR repos, S3 buckets and local files that have been generated.
+
+## Acceptance tests
+
+We use an acceptance test suite to judge releasability of the system. These tell us whether the system can perform its
+functions in a deployed environment, and help us understand whether changes to the code have increased or decreased the
+performance. The tests sit in the Maven module `system-test/system-test-suite`. There are scripts to run these tests
+under `scripts/test/maven`. We run this test suite nightly.
+
+These tests run in JUnit, and use the class `SleeperSystemTest` as the entry point. That class defines the domain
+specific language (DSL) for working with a deployed instance of Sleeper in JUnit. Please review the comment at the top
+of that class, and look at the tests in that module for examples.
+
+This test suite contains both feature tests, and performance tests which work with a larger bulk of data. By default,
+the suite skips the performance tests. You can run that like this:
```

Comment:
This reads a bit odd following the previous sentence. Maybe something like 
`You can run the default test suite like this:`

Reply:
Thanks, I'll change it.